recipe/meta.yaml

```diff
@@ -29,13 +29,14 @@ requirements:
     - r-rcpp >=0.12.15  # [build_platform != target_platform]
   host:
     - r-base
-    - tiledb >2.0
+    - tiledb 2.7.*
```

Comment:
The most recent release is 2.8.* though.  We have not released the R package against 2.7.*.

Reply:
0.11.1 build 0 was linked against 2.7 here on conda, since we previously didn't pin tiledb, it got picked up in the 0.11.1 since conda had 2.7 installed on it. So I decided to keep it with 2.7. Should have made that clear.

Reply:
Can you not force upgrade to 2.8?  That is what package managers with control over versions should provide.

Reply:
Anyway, the setup is in fact very flexible and any version from 2.0 on to 2.8 and dev will build.  I am merely surprised that in something we officially communicate as a build for our users you decide to move away from the pin preference expressed in the source package.

Reply:
Ideally I'd like to keep conda version tied to the same core version we use on main package (cran/source build with prebuilt tiledb). The `>2.0` pin in this package originally meant it was "incorrectly" grabbing whatever was the latest at build time. It worked because the TileDB-R is very well written for maximum compatibility :). Also in general, TileDB-R package updates have come in step with core versions and we've only overlapped with patch version on either package. TileDB 2.7 changed that slightly because of the delays and issues caused by the memory leak in that package. Thus when TileDB 2.7 was released onto conda, this 0.11.1 which released against 2.6.4 for CRAN, picked up the 2.7 release on conda. So 0.11.1 on conda != 0.11.1 on CRAN/source from a libtiledb linkage.

This PR attempts to rectify this by moving to specific pins for TileDB. I specifically choose to use 2.7 only because the existing 0.11.1 on conda has 2.7 linkage. So this is a "noop" from changing the current version built, but fixes the build process for the future. I thought better to keep conda the same for this particular version. Once we have a 0.12.0 tag on github we can _immediately_ move the conda package to 0.12.0 and update the pin to 2.8 and going forward properly keep everything in lock step with what cran linkages between TileDB and TIleDB-R are.

Reply:
Right, I forgot the 2.6.4 aspect.  And yes, I also suspected that you might roll it forward 2.8 once available. It's good to have to ability to create incremental builds.